feature/experimental/gribi/ate_tests/route_removal_during_failover_test/route_removal_during_failover_test.go

```diff
@@ -0,0 +1,609 @@
+
+	entriesBefore := checkNIHasNEntries(ctx, client, *deviations.DefaultNetworkInstance, t)
+
+	// Concurrently run switchover and gribi route flush
+	t.Log("Execute gRIBi flush and master switchover concurrently")
+	go func(msg string) {
+		if _, err := gribi.Flush(client, eID, *deviations.DefaultNetworkInstance); err != nil {
+			t.Logf("Unexpected error from flush, got: %v", err)
+		}
+	}("gRIBi Flush")
+
+	go func(msg string) {
+		switchoverResponse, err := gnoiClient.System().SwitchControlProcessor(context.Background(), switchoverRequest)
+		if err != nil {
+			t.Logf("Failed to perform control processor switchover with unexpected err: %v", err)
+		}
+		t.Logf("gnoiClient.System().SwitchControlProcessor() response: %v, err: %v", switchoverResponse, err)
+	}("Master Switchover")
+
+	t.Log("Switchover/Flush is completed, validate switchoverStatus now")
+
+	primaryAfterSwitch := validateSwitchoverStatus(t, dut, secondaryBeforeSwitch)
+
+	validateSwitchoverTelemetry(t, dut, primaryAfterSwitch)
+
+	// Following reconnection of the gRIBI client to a new master supervisor,
+	// validate if partially deleted entries of IPBlock1  are not present in the FIB
+	// using a get RPC.
+	// Connect gRIBI client to DUT referred to as gRIBI - using PRESERVE persistence and
+	// SINGLE_PRIMARY mode, with FIB ACK requested. Specify gRIBI as the leader.
+	// Check vars for WithInitialElectionID
+
+	t.Log("Reconnect gRIBi client after switchover on new master")
+	client.Connection().WithStub(gribic).WithPersistence().WithInitialElectionID(eID.Low, eID.High).
+		WithFIBACK().WithRedundancyMode(fluent.ElectedPrimaryClient)
+
+	client.Start(ctx, t)
+	defer client.Stop(t)
+	defer func() {
+		// Flush all entries after test.
+		if err := gribi.FlushAll(client); err != nil {
+			t.Error(err)
+		}
+	}()
+
+	// Reconnect gribi client
+	client.StartSending(ctx, t)
+
+	if err := awaitTimeout(ctx, client, t, time.Minute); err != nil {
+		t.Log("Try to connect gRIBi client again, retrying...")
+		client.Connection().WithStub(gribic).WithPersistence().WithInitialElectionID(eID.Low, eID.High).
+			WithFIBACK().WithRedundancyMode(fluent.ElectedPrimaryClient)
+		client.Start(ctx, t)
+		client.StartSending(ctx, t)
+		if err := awaitTimeout(ctx, client, t, time.Minute); err != nil {
+			t.Fatalf("Await got error during session negotiation for client: %v", err)
+		}
+	}
+
+	t.Log("Compare route entries after switchover")
+	gr, err = args.client.Get().
+		WithNetworkInstance(*deviations.DefaultNetworkInstance).
+		WithAFT(fluent.IPv4).
+		Send()
+	if err != nil {
+		t.Fatalf("got unexpected error from get, got: %v", err)
+	}
+
+	prefixEntryList := []string{}
+	for _, r := range gr.GetEntry() {
+		entry := strings.Split(r.GetIpv4().Prefix, "/")
+		prefixEntryList = append(prefixEntryList, entry[0])
+	}
+
+	if len(prefixEntryList) == len(virtualIPs) {
+		t.Error("After switchover, on new master seeing unexpected number of route entries")
```

Comment:
why have two t.Error() here? why can't them be combined into one?

Reply:
updated.